src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -101,7 +101,7 @@ protected void BuildFieldType(EventStreamFieldType fieldType, MemberInfo memberI
                 var resolver = new MemberResolver(propertyInfo, BuildMemberInstanceExpression(memberInfo));
                 fieldType.Arguments = null;
                 fieldType.Resolver = resolver;
-                fieldType.AsyncSubscriber = null;
+                fieldType.Subscriber = null;
```

Comment:
I see xml comments mentioning `AsyncSubscriber`.

Reply:
I found & fixed one but couldn't find any others.  Lmk if you find any.